plugin/template/README.md

```diff
@@ -63,7 +84,7 @@ The `.invalid` domain is a reserved TLD (see [RFC-2606 Reserved Top Level DNS Na
 . {
     proxy . 8.8.8.8
 
-    template ANY ANY "[.]invalid[.]$" {
+    template ANY ANY invalid. {
```

Comment:
Should we make this final dot implicit?

Reply:
Good question, I'm using the same logic as etcd.... And I'm usually trying to mark qualified names as such.

Related: If nothing is specified I'm right now matching '.' - is there a reason to switch to the etcd/setup.go method of
```
		if len(etc.Zones) == 0 {
			etc.Zones = make([]string, len(c.ServerBlockKeys))
			copy(etc.Zones, c.ServerBlockKeys)
		}
```

And it looks like I should normalize, too (according to the etcd source)

Reply:
yes, this should be `Normalized`

Reply:
drop the final dot (as this is not needed anymore)